bears/haml/HamlLintBear.py

```diff
@@ -0,0 +1,30 @@
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+
+from dependency_management.requirements.GemRequirement import GemRequirement
```

Comment:
Just the order had to be changed so it looks good now. We don't need to leave a line however. :wink: